### ISSUE TITLE
Missing Windows uninstallers

### ISSUE THREAD

**Issue by massic80** [supertuxkart/stk-code#4580]
## Description
I installed 1.0.0, then 1.1.0, then 1.2.0.
When I noticed that older versions were still on my disk, I attempted to remove them, but they are lacking an uninstaller.
Were they missing from the beginning? Were they removed by 1.2.0 installer?
Is it correct to remove them by pruning their folder, or shall I be _more delicate_?

## Configuration
STK release version: 
1.2.0

STK Source (PPA, distribution package, official binary, etc.): 
Official binary

System: 
Windows 10

**Comment by Benau** [comment]
There are uninstallers for every stk release, try find the uninstall.exe from program file / supertuxkart if you dont find it in control center

**Comment by massic80** [comment]
Nah, no uninstallers in STK 1.0 and 1.1 directories. A Recuva scan didn't find any deleted file, but this doesn't mean anything particular.
Did something change in the installer? Did previous folders have the uninstaller set elsewhere?
![stk100](https://user-images.githubusercontent.com/6404170/129730001-6b5ec9b1-8e19-4505-b111-ae67d1bb6e2f.png)
![stk110](https://user-images.githubusercontent.com/6404170/129730004-edbdcb7b-1c9a-48b8-a638-b168f5c56373.png)
![stk120](https://user-images.githubusercontent.com/6404170/129730006-663af105-f2ce-4a79-923f-a2d64eaaac03.png)

**Comment by Benau** [comment]
No much idea, you can just remove 1.0 and 1.1 folder manually

For every release installer we did make sure it contains a proper uninstaller though

**Comment by massic80** [comment]
OK, thank you.
I was just concerned about registry keys: is cutting the folder away enough?
What if I tried reinstalling legacy versions and checking whether the uninstallers reappear? :)

**Comment by Benau** [comment]
Only registry key used is for uninstaller:
https://github.com/supertuxkart/stk-code/blob/6d29bb21ba3cb99f1257cd506bb093c2be33e9c5/tools/windows_installer/supertuxkart-github-actions.nsi#L255

If you don't see uninstaller in control center then maybe it's gone already

**Comment by massic80** [comment]
OK, thank you :)
I'll let you know whether 1.3.0 will remove 1.2.0' installer ;)

**Comment by massic80** [comment]
Micro-sidenote: reinstalling 1.0 and 1.1 I got the uninstallers back (in a new directory, or they couldn't be installed) and could uninstall as usual.
